js/script.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+
+var output = document.getElementById('output');
+var outputinfo = document.getElementById('outputinfo');
+var buttonC = document.getElementById('buttonC');
+var buttonF = document.getElementById('buttonF');
+
+output.innerHTML = 'Wcisnij guzik jeśli chcesz zamieić stopnie' + '<br><br>'; 
+
+buttonC.addEventListener('click', function() {	  
+    var degreesC = window.prompt('Prosze podać stopnie celcjusza');
+    var valueC = parseFloat(degreesC);
+    var degreesF = convertCelsiusToFeranheit(degreesC);
+
+    if (!isNaN(valueC)) {
+            output.innerHTML = 'Stopnie Celsjusza: ' + degreesC + '&#x2103 ' + '<br>' + 'Stopnie Fahrenheita: ' + degreesF + '&#x2109' + '<br>'; 
+    } else if (isNaN(valueC)) {
+            output.innerHTML = 'Nie wpisałeś cyfry';
+    } else {
+        output.innerHTML = 'Nic nie wpisałeś';
+    }  
+    changeStateOfWater(valueC);
+    stateOfDressToTemperature(valueC);
+});
+
+buttonF.addEventListener('click', function() {	   
+    var degreesF = window.prompt('Prosze podać stopnie Fahrenheita');
+    var valueF = parseFloat(degreesF);
+    var degreesC = convertFeranheitToCelsius(degreesF);
+    
+    if (!isNaN(valueF)) {
+            output.innerHTML = 'Stopnie Fahrenheita: ' + degreesF + '&#x2109 ' + '<br>' + 'Stopnie Celsjusza: ' + degreesC + '&#x2103' + '<br>'; 
+    } else if (isNaN(valueF)) {
+            output.innerHTML = 'Nie wpisałeś cyfry';
+    } else {
+        output.innerHTML = 'Nic nie wpisałeś';
+    }  
+    var celcius = convertFeranheitToCelsius(valueF);
+    changeStateOfWater(celcius); 
+    var dress = convertFeranheitToCelsius(valueF);
```

Comment:
`dress` i `celcius` zawsze będą miały taką samą wartość - policz to raz i wykorzystaj w obu miejscach